.pylintrc

```diff
@@ -0,0 +1,549 @@
+[MASTER]
```

Comment:
So I generated a default pylintrc file with the following:

```pylint --generate-rcfile > ~/.pylintrc```

I then edited that one setting which was to do with using uppercase for constants. 

Was this not the best way of doing this?

Reply:
it sounds like this method results in the desired behaviour, so it's correct.

however by including all of the possible config it is harder to see what has changed vs the default behaviour (which becomes more relevant as you become familiar with the tool and how it usually behaves).

the same outcome would be achieved by only including the one line you mentioned and it would be clearer at a glance how our config is different to the default behaviour.

Reply:
Ah I didn't realise that if you don't define any value in the pylint configuration, that it'll use its default value.

I'll strip this back to just the singular rule we're interested in:
```const-naming-style=any```

Reply:
this is a fairly common pattern for config files - there will be defaults which can be overriden in different ways, and often an order of precedence. e.g. defaults are overriden by config file(s), which is overridden with env vars, which is overriden by cli flags.

Reply:
e.g. https://www.terraform.io/docs/configuration/variables.html#variable-definition-precedence